### PR TITLE
Cosmetic change to pihole -v -c output

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
This has been doing my head in every time I update. Before:

image

After:
image

### CODE REVIEW

File: advanced/Scripts/version.sh [hunk 1]
```diff
@@ -153,7 +153,7 @@ versionOutput() {
     if [[ -n "$current" ]] && [[ -n "$latest" ]]; then
         output="${1^} version is $branch$current (Latest: $latest)"
     elif [[ -n "$current" ]] && [[ -z "$latest" ]]; then
-        output="Current ${1^} version is $branch$current."
+        output="Current ${1^} version is $branch$current"
```
Comment (dschaper) [inline advanced/Scripts/version.sh:156]:
```suggestion
        output="Current ${1^} version is $branch$current 🚀"
```

**Review by dschaper — APPROVED** [review]

**Comment by dschaper** [comment]
Did we add/remove Stickler?

File: advanced/Scripts/version.sh (hunk 1 above)
Comment (PromoFaux) [reply advanced/Scripts/version.sh:156]: Tempting...

**Comment by PromoFaux** [comment]
I don't think we removed it, I noticed once before it got stuck but I just ignored it with view to looking into it "later". I guess later never came ;)

**Pull request merged**